redGreen.js

```diff
@@ -1,23 +1,28 @@
 import fetch from 'node-fetch';
 
+// GETTING NAMES FROM OBJECTS
 const getName = ({ name }) => {
   return name;
 };
 
 
+// COPYING AND PUSHING ITMES TO ARRAYS
 const copyAndPush = (arr, nums) => {
   const newArr = [...arr];
   newArr.push(nums);
   return newArr; 
 };
 
+
+// CAPITALIZING AND FILTERING WORDS
 const capitalizeAndfilter = (arr) => {
   const newSD = arr.map(item => item.toUpperCase());
   return newSD.filter(word => word[0] === 'F');
```

Comment:
This function needs to remove words starting with F from the array and return the array without those words. You'll want to adjust your test and make sure you're adding more to the function here to create a new array and push anything that doesn't start with an F into that array, then return that one instead of the original.